AppController/appcontroller.cfg

```diff
@@ -1,5 +0,0 @@
-check process controller-17443 matching "/usr/bin/ruby -w /root/appscale/AppController/djinnServer.rb"
```

Comment:
This file is not used anylonger to the best of my knowledge.


On Thu, 2016-06-16 at 16:06 -0700, Meni Vaitsi wrote:
> In AppController/appcontroller.cfg:
> > @@ -1,5 +0,0 @@
> > -check process controller-17443 matching "/usr/bin/ruby -w
> /root/appscale/AppController/djinnServer.rb"
> Remind me, why we removed this?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.